R/create_template.R

```diff
@@ -340,14 +402,38 @@ create_template <- function(
         }
       }
 
+      # run satf::exp_all_figs_tables() if rda files not premade
```

Comment:
needs check for when users run create_template with no argument - this will probably break if they do so (maybe it doesn't though and you've addressed this). 

Maybe good to also add a switch like with convert_output to run this or not but set it TRUE by default

Reply:
Yes, definitely make this workable when create_template has no args. Great catch. Will add now